setup/nextcloud.sh

```diff
@@ -367,6 +394,29 @@ cat > /etc/cron.d/mailinabox-nextcloud << EOF;
 EOF
 chmod +x /etc/cron.d/mailinabox-nextcloud
 
+# Set up a cron job for Calendar updates, per advice in the Nextcloud docs
+# https://docs.nextcloud.com/server/24/admin_manual/groupware/calendar.html#background-jobs
+cat > /etc/cron.d/mailinabox-nextcloud-dav << EOF;
+#!/bin/bash
+# Mail-in-a-Box
+*/5 * * * *	root	sudo -u www-data php$PHP_VER -f /usr/local/lib/owncloud/occ dav:send-event-reminders
```

Comment:
Do we need another file? Can't it be added to `/etc/cron.d/mailinabox-nextcloud` ?

Reply:
Ok, I've added this new cron job to /etc/cron.d/mailinabox-nextcloud instead.

No, that command didn't generate any output on my side, so leaving as it:

```
vagrant@mailinabox:/usr/local/lib/owncloud$ sudo -u www-data php8.0 ./occ dav:send-event-reminders
vagrant@mailinabox:/usr/local/lib/owncloud$ 
```